server/http_handler.go

```diff
@@ -37,6 +37,7 @@ import (
 	"github.com/pingcap/parser/terror"
 	"github.com/pingcap/tidb/config"
 	"github.com/pingcap/tidb/domain"
+	"github.com/pingcap/tidb/domain/util"
```

Comment:
I want to use this package in package `inforschema`, but this package name implies it's only used by `domain`, it is a little bit confusing to me.

Reply:
Some pkg like `github.com/pingcap/tidb/ddl/util` also not only use in `ddl` pkg.

Move all `xxx/util` pkg to `util` directory maybe make the util directory complexity and unwieldy.

Reply:
Fair enough.